AxonDeepSeg/integrity_test.py

```diff
@@ -35,13 +37,14 @@ def integrity_test():
         with open(path_configfile, 'r') as fd:
             config_network = json.loads(fd.read())
 
+
         # Launch the axon and myelin segmentation on test image sample provided in the installation
         print('Computing the segmentation of axon and myelin on test image.')
-        prediction = axon_segmentation([path_testing], ["image.png"], path_model, config_network, prediction_proba_activate=True, verbosity_level=4)
+        prediction = axon_segmentation([path_testing], [image_name], path_model, config_network, prediction_proba_activate=True, verbosity_level=4)
 
         # Read the ground truth mask and the obtained segmentation mask
         mask = ads.imread(path_testing / 'mask.png')
-        pred = ads.imread(path_testing / 'AxonDeepSeg.png')
+        pred = ads.imread(path_testing / (image_name[:-4] + axonmyelin_suffix.name))
```

Comment:
Getting the filename without the extension this way is not recommended. If image_name is a Path() object, then it can simply be fetched using `.stem`:

```python
(base) mathieuboudreau@mathieu ~ % python
Python 3.7.6 (default, Jan  8 2020, 13:42:34) 
[Clang 4.0.1 (tags/RELEASE_401/final)] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from pathlib import Path
>>> filename = Path("image.png")
>>> print(filename.stem)
image
>>> 

```

Reply:
Agreed, will make changes.

Reply:
Resolved in commit 90e53b5